packages/admin/admin-stories/src/docs/components/DateAndTimePickers/04_DateRangePicker.stories.mdx

```diff
@@ -0,0 +1,33 @@
+import { Meta, Canvas, Story, Source } from "@storybook/addon-docs";
+import DatePickerLocalisation from "./common/DatePickerLocalization.mdx";
+
+<Meta title="Docs/Components/Date & Time Pickers/Date-Range Picker" />
```

Comment:
I think that the Date & Time Pickers rather belong to the Form section:

`Docs/Form/Components/Date & Time Pickers/Date-Range Picker`

Reply:
True 👍🏼